src/realm.js

```diff
@@ -660,40 +657,31 @@ export class Realm {
     return new Constructor(this, types, values, args, buildNode, { kind, intrinsicName });
   }
 
-  rebuildObjectProperty(object: Value, key: string, propertyValue: Value, path: string) {
-    if (!(propertyValue instanceof AbstractValue)) return;
-    if (!propertyValue.isIntrinsic()) {
-      propertyValue.intrinsicName = `${path}.${key}`;
-      propertyValue.args = [object];
-      propertyValue._buildNode = ([node]) => t.memberExpression(node, t.identifier(key));
-      this.rebuildNestedProperties(propertyValue, propertyValue.intrinsicName);
-    }
-  }
-
-  rebuildNestedProperties(abstractValue: AbstractValue | UndefinedValue, path: string) {
-    if (!(abstractValue instanceof AbstractObjectValue)) return;
-    let template = abstractValue.getTemplate();
-    invariant(!template.intrinsicName || template.intrinsicName === path);
-    // TODO #882: We are using the concept of "intrinsic values" to mark the template
```

Comment:
This old TODO comment still have some relevance: In some cases, it's important to associate the intrinsic object with a particular point in time that the serializer should know about. In other words, `intrinsicNameGenerated` are a strange thing.